### DISCUSSION TITLE
Different theme for signed-in vs signed-out

### DISCUSSION THREAD

**Discussion by frankprogrammer** [monkeytypegame/monkeytype#5225] [Ideas]
There has been several times where I don't notice that my account is signed out and I end up losing a day of stats, including my streak. It would be helpful if the current theme setting was separate between the logged-out and logged-in states. I could then set them to be different and I would more likely notice that I was signed out.

**Comment by sanidhyas3s** [comment]
Currently if you're logged out the theme just carries over to the logged out session, and not the other way around. It could be that a logged out session starts from the default theme rather than carrying over but I guess @Miodec made that decision keeping something in mind. Though the idea is good, there is already a `Sign in to save your result` message on the results page which serves exactly this purpose.
